results/README.md

```diff
@@ -8,4 +8,4 @@ record here.
 
 | directory | commit No. | evaluation | estimators | report link |
 |-|-|-|-|-|
```

Comment:
Can you add the instruction of viewing html directly from github?

Reply:
Ack. I will do in a separate PR.